app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -57,6 +63,9 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
   @Inject
   lateinit var userClock: UserClock
 
+  @field:[Inject Named("training_video_youtube_id")]
```

Comment:
Curious,

Wouldn't
```kotlin
@Inject
@Named("training_video_youtube_id")
lateinit var youTubeVideoId: String 
```

suffice?

Reply:
This doesn't work in Kotlin.
For field injection, we need to specify `@field` annotation because a property in Kotlin is backed by `set()` and `get()` and Dagger expects a field and not a property.
Read [here](https://medium.com/@WindRider/correct-usage-of-dagger-2-named-annotation-in-kotlin-8ab17ced6928)